doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1274,24 +1280,30 @@ Managing the local context
    .. exn:: The variable @ident is already defined.
       :undocumented:
 
-   .. tacv:: set (@ident := @term) in @goal_occurrences
+   .. tacv:: set (@ident := @term) in @occurrences
+
+      .. insertprodn occurrences occurrences
+
+      .. prodn::
+         occurrences ::= {+ @int }
+         | @ident
```

Comment:
This is wrong. I just tested and `set` supports the `in_clause`. Example:

```
Goal 0 + 0 = 0 + 0.
set (n := 0) in * |- *.
```

Reply:
I'm going to back out changing goal_occurrences to occurrences.  Too many cascading changes to completely fix the many incorrect tactics that referenced goal_occurrences.

Reply:
OK